homeassistant/util/color.py

```diff
@@ -528,6 +528,16 @@ def color_temperature_to_rgb(
     return red, green, blue
 
 
+def color_temperature_to_rgbww(
+    temperature: int, brightness: int, min_mireds: int, max_mireds: int
+) -> tuple[int, int, int, int, int]:
+    """Convert color temperature to rgbcw."""
+    max_min_temp_diff = max_mireds - min_mireds
+    warm = ((max_mireds - temperature) / max_min_temp_diff) * brightness
```

Comment:
Let's use same naming as in other functions:
```suggestion
    mired_range = max_mireds - min_mireds
    warm = ((max_mireds - temperature) / mired_range) * brightness
```

Reply:
Thanks.